controller/auth-controller.js

```diff
@@ -0,0 +1,66 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const user = require("../models/user");
+
+exports.login = async (req, res) => {
+  let name = req.body.name;
```

Comment:
Always use `const` if you're not reassigning to the same variable name.

Reply:
Okay